src/Microsoft.DotNet.ProjectModel/ProjectContextBuilder.cs

```diff
@@ -191,17 +165,8 @@ public ProjectContext Build()
 
             RootDirectory = GlobalSettings?.DirectoryPath ?? RootDirectory;
             PackagesDirectory = PackagesDirectory ?? PackageDependencyProvider.ResolvePackagesPath(RootDirectory, GlobalSettings);
-
-            FrameworkReferenceResolver frameworkReferenceResolver;
-            if (string.IsNullOrEmpty(ReferenceAssembliesPath))
-            {
-                // Use the default static resolver
-                frameworkReferenceResolver = FrameworkReferenceResolver.Default;
-            }
-            else
-            {
-                frameworkReferenceResolver = new FrameworkReferenceResolver(ReferenceAssembliesPath);
-            }
+            ReferenceAssembliesPath = ReferenceAssembliesPath ?? FrameworkReferenceResolver.GetDefaultReferenceAssembliesPath();
+            var frameworkReferenceResolver = new FrameworkReferenceResolver(ReferenceAssembliesPath);
```

Comment:
Bad Merge? We want to share framework resolver instances as much as possible to reduce I/O reading RedistList files